Tasks/merge.js

```diff
@@ -1,19 +1,6 @@
+'use strict'
 // Merge two dictionaries
 
-merge_two_objects=(function merge_two_objects(object_1,object_2){
-    object_3 = [({})][0];
-   for (attribute_name in object_1) {
-       (
-          object_3[attribute_name] == object_1[attribute_name]
-       );
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const merge = (objOne, objTwo) => Object.assign({}, objOne, objTwo);
```

Comment:
Implement this with for loops, don't use Object.assign

Reply:
https://github.com/HowProgrammingWorks/Exams/pull/61/commits/7d5678bdb71c0625b62a2e410f942dbd6817cb50

Reply:
https://github.com/VeSpy4u/Exams/commit/6ea39056ddc91ac9db331989a2f24de6fc2a50e5#diff-583e7a1e779616b1571de750b4b0efa988eb1059b763e3fa4402f57a61ed9609

Reply:
https://github.com/VeSpy4u/Exams/commit/6ea39056ddc91ac9db331989a2f24de6fc2a50e5